web/server/entrypoint_test.go

```diff
@@ -23,48 +29,80 @@ import (
 // expected builtin resources.
 const numResources = 20
 
-func TestNumResources(t *testing.T) {
+func TestEntrypoint(t *testing.T) {
 	if runtime.GOARCH == "arm" {
 		t.Skip("skipping on 32-bit ARM, subprocess build warnings cause failure")
 	}
-	logger := logging.NewTestLogger(t)
-	cfgFilename := utils.ResolveFile("/etc/configs/fake.json")
-	cfg, err := config.Read(context.Background(), cfgFilename, logger)
+	serverPath, err := testutils.BuildTempModule(t, "web/cmd/server/")
 	test.That(t, err, test.ShouldBeNil)
 
-	p, err := goutils.TryReserveRandomPort()
-	test.That(t, err, test.ShouldBeNil)
+	t.Run("number of resources", func(t *testing.T) {
```

Comment:
Moved to a shared test to reuse the `BuildTempModule` call which takes 3 seconds on my local machine.